package.json

```diff
@@ -18,6 +18,7 @@
     "publish-shell": "./shell/scripts/publish-shell.sh",
     "clean": "./scripts/clean",
     "lint": "./node_modules/.bin/eslint --max-warnings 0 --ext .js,.ts,.vue .",
+    "lint-l10n": "./node_modules/.bin/yamllint shell/assets/translations",
```

Comment:
This may just be pedantic, but would pointing to `shell/assets/translations` rather than `./shell` cause issues?